### ISSUE TITLE
Port to Java 9+

### ISSUE THREAD

**Issue by codeanticode** [processing/processing-android#558]
Processing 4 will be using OpenJDK 9+, so the next release of the mode should be made compatible with it. However, seems that the Android SDK tools are not yet fully compatible with Java 9 and higher:

https://stackoverflow.com/questions/46402772/failed-to-install-android-sdk-java-lang-noclassdeffounderror-javax-xml-bind-a

More on this topic:

https://stackoverflow.com/questions/53076422/getting-android-sdkmanager-to-run-with-java-11/55982976

In conclusion, seems that for the SDK tools to work with Java 9 and higher, the classpath in the sdkmanager and advmanager need to include the jars from [JAXB](https://javaee.github.io/jaxb-v2/) and [istack-commons-runtime](https://mvnrepository.com/artifact/com.sun.istack/istack-commons-runtime).

**Comment by codeanticode** [comment]
All needed bits to support OpenJDK 9+ are in place now in the [android-openjdk](https://github.com/processing/processing-android/tree/android-openjdk) (and bumped version to 5.0-alpha0 branch). Tested on AdoptOpenJDK 11.

Until the Android SDK supports OpenJDK 9+, the mode will [automatically add](https://github.com/processing/processing-android/commit/195b21b801331ae689e6e6e5a25d6d8f46897c14) the JAXB and istack jars to the classpath of the avd and sdk manager tools (only when the user downloads a SDK for exclusive use with the mode, otherwise it simply shows a message explaining the incompatibility between Android SDK and Java 9+)
